learningeconomy/LearnCard
# [WE-2603] New VC display

[![workerB](https://img.shields.io/endpoint?url=https%3A%2F%2Fworkerb.linearb.io%2Fv2%2Fbadge%2Fprivate%2FU2FsdGVkX18Wj8oVrMTOOkScrtV6mRLX3htH4bfaabk%2Fcollaboration.svg%3FcacheSeconds%3D60)](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=vFbxbEh)
# Overview

#### 🎟 Relevant Jira Issues

[WE-2603](https://welibrary.atlassian.net/browse/WE-2603)

#### 📚 What is the context and goal of this PR?
We've got a new design for VC's! [Figma](https://www.figma.com/file/RCbnVvxZtrgWTIkelNpepG/Learn-Card-Wallet?node-id=3137%3A21101&t=5pOqTHEUYk2GLJgI-0)

This also has code for handling skills and a notes field, but those things are commented out right now since we're not using them

#### 🥴 TL; RL:

#### 💡 Feature Breakdown (screenshots & videos encouraged!)
With defaults:
![image](https://user-images.githubusercontent.com/63376352/219224386-67c4a030-9dfa-4b8f-a5e3-d41cbcf2961b.png)
![image](https://user-images.githubusercontent.com/63376352/219224602-4a34ba17-1bfb-4fdb-9309-4af79f518141.png)

Attachments:
![image](https://user-images.githubusercontent.com/63376352/219226049-a788180e-a796-4ef6-901e-312e012a2f17.png)
(When an attachment is clicked it will be handled by a custom function passed in by the consumer)

Credential Verifications:
![image](https://user-images.githubusercontent.com/63376352/219226234-ffa2f4a9-dbc1-4623-987c-1e406bf5a43d.png)

You can also customize many of the elements:
![image](https://user-images.githubusercontent.com/63376352/219225119-4fc7cac1-8a8e-437b-a36d-4bb33a1d9d94.png)
- The X appears if you specify a `onXClick` function (expected value is something like `closeModal`)
- The background can be set to a custom image or color
- The issuer and issuee image and name can be overridden
- The bottom right icon and background color can be customized

#### 🛠 Important tradeoffs made:

#### 🔍 Types of Changes


#### 💳 Does This Create Any New Technical Debt? ( If yes, please describe and [add JIRA TODOs](https://welibrary.atlassian.net/jira/software/projects/WE/boards/2) )

# Testing

#### 🔬 How Can Someone QA This?

 - Start storybook and check out the stories for VCDisplayCard2 and VCCard
 - That should really cover it, but feel free to
   - mess with the props in storybook or modify the credentials in `test.helpers.ts`
   - import this component and use it in a different context


#### 📱 🖥 Which devices would you like help testing on?


#### 🧪 Code Coverage

Storybook

# Documentation

#### 📜 Gitbook


#### 📊 Storybook



# ✅ PR Checklist

### 🚀 Ready to squash-and-merge?:
